src/VisualStudio/Core/Def/Implementation/TaskList/ExternalErrorDiagnosticUpdateSource.cs

```diff
@@ -530,6 +531,36 @@ private bool IsLive(Project project, Compilation compilation, DiagnosticData dia
                 }
 
                 return false;
+
+                bool IsDocumentLevelDiagnostic(DiagnosticData diagnoaticData)
```

Comment:
1. spelling: `diagnoatic`
2. can we make this a static local function?  i far prefer those as it makes it clear it can't be capturing/mutating local state.  Thanks!

Reply:
sure